definition/address-book/v3/Address.ts

```diff
@@ -0,0 +1,22 @@
+import { Position } from '../../Position';
+import Contact from './Contact';
+import Score from './Score';
+
+type Address = {
+  id: string,
+  alias?: string,
+  street: string,
+  additional_street?: string,
+  city: string,
+  zip_code?: string,
+  province?: string,
+  country: string,
+  position: Position,
+  timezone: string,
+  contacts: Contact[],
+  scores: Score[],
+  created_at: string,
+  updated_at: string
+};
```

Comment:
Probably better to extend `InputAddress`

Reply:
Also not really sure about `zip_code` optional. I know that in the BRD it is listed as optional, but in address-book it was required and we have it required in `uploadAddress` logic. 
Might be worth to ask archi about this one, in case they made mistake

Reply:
I'd rather not, because of the id (which is optional in Input), but also because both will be used in different contexts

Reply:
After extend you can override field behavior just fine. 
```suggestion
type Address = InputAddress & {
  id: string,
};
```
If `InputAddress` has `id` set as optional, it will be required in `Address`.

About `zip_code`, that's my point in legacy data it was required(you cannot create a record without it), so I was surprised it's not the case here.